src/main/java/org/jenkinsci/plugins/android_lint/LintPublisher.java

```diff
@@ -96,23 +112,32 @@ public String getPattern() {
         return pattern;
     }
 
+    /**
+     * Sets the Ant file-set pattern of files to work with.
+     */
+    @DataBoundSetter
+    public void setPattern(final String pattern) {
+        this.pattern = pattern;
+    }
```

Comment:
Is there is a reason to add this method, rather than just setting `@DataBoundSetter` on the `pattern` field?
Is it required for setting options via Workflow?